omod/src/main/webapp/resources/htmlforms/018-HTSClientCard.xml

```diff
@@ -320,405 +320,411 @@
+                                        <div class="col-md-3">
+                                            <obs conceptId="165169" answerConceptId="165126"
+                                                 answerLabel="4 - Migrant workers" style="checkbox"/>
+                                        </div>
+                                    </div>
+                                    <div class="row ARTSpecialCateogryEnableDisable1">
+                                        <div class="col-md-3">
+                                            <obs conceptId="165169" answerConceptId="162198"
+                                                 answerLabel="5 - Truckers" style="checkbox"/>
+                                        </div>
+                                        <div class="col-md-3">
+                                            <obs conceptId="165169" answerConceptId="159674"
+                                                 answerLabel="6 - Fisher Folks" style="checkbox"/>
+                                        </div>
+                                        <div class="col-md-3">
+                                            <obs conceptId="165169" answerConceptId="165127"
+                                                 answerLabel="7 - Refugees" style="checkbox"/>
+                                        </div>
+                                        <div class="col-md-3">
+                                            <obs conceptId="165169" answerConceptId="90041"
+                                                 answerLabel="8 - Pregnant" style="checkbox"/>
+                                        </div>
+                                    </div>
+                                    <div class="row ARTSpecialCateogryEnableDisable1">
+                                        <div class="col-md-3">
+                                            <obs conceptId="165169" answerConceptId="99601"
+                                                 answerLabel="9 - Breastfeeding" style="checkbox"/>
+                                        </div>
+                                        <div class="col-md-3">
+                                            <obs conceptId="165169" answerConceptId="165132" answerLabel="10 - AGYW"
+                                                 style="checkbox"/>
+                                        </div>
+                                        <div class="col-md-3">
+                                            <obs conceptId="165169" answerConceptId="162558"
+                                                 answerLabel="11 - People with Disabilities" style="checkbox"/>
+                                        </div>
+                                        <div class="col-md-3">
+                                            <obs conceptId="165169" id="165169" answerConceptId="90002"
+                                                 answerLabel="12 - Others (specify)" style="checkbox"/>
```

Comment:
Remove (specify)  from the "12 - Others" checkbox option and add it as a label on the other specify text box